query/errors.go

```diff
@@ -43,4 +43,13 @@ func (errors SyntaxErrors) Error() string {
 	return strings.Join(errorStrings, "\n")
 }
 
+// EmptyAggregateError is an Error for attempts to aggregate empty SeriesLists.
+type EmptyAggregateError struct {
```

Comment:
Since there's nothing special for this error, you can just create it using errors.New()